deploy/docker-compose/README

```diff
@@ -0,0 +1,153 @@
+# MONAI Deploy Lite
+
+MONAI Deploy Lite provides a simple & lightweight solution for running MONAI Applications (MAPs) on a minimal MONAI Deploy platform.
+
+It is meant to bridge the big gap between local development of MAPs in a laptop using the MONAI Deploy App SDK, and a production-ready deployment. 
+
+Users of MONAI Deploy Lite will be able to run their MAPs, connected to a test PACS, or their own test / research PACS, for further validation, confidently taking steps towards production.   
+
+By reusing the same key core services for DICOM I/O and AI workflow orchestration, it provides the same functionality and consistent experience independently on where and how the applications are run, with minimal changes for the end user.
+
+
+## Prerequisites
+
+- [Docker Engine](https://docs.docker.com/engine/install/) v20.10.18+
+- [Docker Compose Plug-in](https://docs.docker.com/compose/install/) v2.10.2+
+- MONAI Deploy MAPs built using the [MONAI Deploy App SDK](https://github.com/Project-MONAI/monai-deploy-app-sdk)
```

Comment:
Other Docker/OCI containers are supported too! But limited to one Docker only, for each task run by the Task Docker plugin, correct?

Reply:
Yes.